src/test/test_TriangleCount.c

```diff
@@ -482,7 +482,7 @@ TEST_LIST = {
     {"TriangleCount_Methods6", test_TriangleCount_Methods6},
     {"TriangleCount"         , test_TriangleCount},
     {"TriangleCount_many"    , test_TriangleCount_many},
-    {"TriangleCount_autosort", test_TriangleCount_autosort},
+    // {"TriangleCount_autosort", test_TriangleCount_autosort},
```

Comment:
@mcordery, `TriangleCount_autosort` created a bigger matrix in memory and _I believe_, was the cause of the LC crash reported in #13.